.gitignore

```diff
@@ -27,7 +27,12 @@ tools/Cake
 packages/
 # !packages/repositories.config
 
+# NUnit results
+TestResult.xml
 
+# Tools installed by Cake
+tools/
+!tools/packages.config
```

Comment:
Not sure this line is required - looks like you're using the bootstrapping scripts that download packages.config from Cake?